week-1/E-strings/exercise.js

```diff
@@ -1,3 +1,4 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var message = "My name is Mohamed Ali";
+var messagetype = typeof message;
+console.log(messagetype);
```

Comment:
According to the [instructions](https://github.com/rarmatei/js-exercises/tree/master/week-1/E-strings) you should also log the message itself, so there are two logs.